src/test/java/com/google/api/codegen/gapic/GapicCodeGeneratorTest.java

```diff
@@ -64,19 +67,22 @@ public GapicCodeGeneratorTest(
             null,
             "library",
             null,
+            TransportProtocol.GRPC,
```

Comment:
Could we also add equivalent tests for `HTTP`?

Reply:
This test is for gapic yaml v1 generation, which is not applicable for DIREGAPIC.